index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Leonardo Pareja Portfolio</title>
+</head>
+<body>
+  <header class="block">
+    <h1 class="name">Leonardo Pareja</h1>
+    <img class="menu" src="images/icons/Menu_icon.svg" alt="menu icon">
```

Comment:
- [ ] I think you could use the `<nav>` tag to wrap the hamburger menu icon and also all the links since these will be part of the navigation element :100: